arches/install/arches-templates/tests/test_settings.py-tpl

```diff
@@ -16,37 +16,14 @@ You should have received a copy of the GNU Affero General Public License
 along with this program. If not, see <http://www.gnu.org/licenses/>.
 """
 
-from arches.settings import *
-import arches
 import os
 
-try:
-    from django.utils.translation import gettext_lazy as _
-except ImportError:  # unable to import prior to installing requirements
-    pass
+from {{ project_name }}.settings import *
 
 PACKAGE_NAME = "{{ project_name }}"
-APP_NAME = '{{ project_name }}'
 
-APP_ROOT = os.path.dirname(__file__)
-TEST_ROOT = os.path.normpath(os.path.join(ROOT_DIR, "..", "tests"))
-
-ROOT_URLCONF = '{{ project_name }}.urls'
-
-MIN_ARCHES_VERSION = arches.__version__
-MAX_ARCHES_VERSION = arches.__version__
-
-# LOAD_V3_DATA_DURING_TESTS = True will engage the most extensive the of the v3
-# data migration tests, which could add over a minute to the test process. It's
-# recommended that this setting only be set to True in tests/settings_local.py
-# and run in specific cases at the discretion of the developer.
-LOAD_V3_DATA_DURING_TESTS = False
-
-RESOURCE_GRAPH_LOCATIONS = (os.path.join(TEST_ROOT, "fixtures", "resource_graphs"),)
-
-ONTOLOGY_FIXTURES = os.path.join(TEST_ROOT, "fixtures", "ontologies", "test_ontology")
-ONTOLOGY_PATH = os.path.join(TEST_ROOT, "fixtures", "ontologies", "cidoc_crm")
```

Comment:
`pip install arches` won't install anything from core arches tests, so we shouldn't suggest depending on anything there. (This might happen to work on an editable install and then fail on CI.)